### ISSUE TITLE
Persist `network_id` with `~/.kindelia/state` and check on start

### ISSUE THREAD

**Issue by steinerkelvin** [Kindelia/Kindelia#163]

**Comment by dan-da** [comment]
how should it be persisted?   I'm thinking something like:

~/.kindelia/state/<network_id>/{blocks,heaps}

That way, state for various networks can be stored side by side.   And the active network_id can be selected from the config file.

**Comment by steinerkelvin** [comment]
That's sensible. I was thinking of just storing it and showing an error when the persisted state is incompatible with the current network id and prompting the user to remove it.

That's more useful — much easier to switch between networks. We can even have config like this:

```
[node.network]
network_id = "0xCAFE0004"

[node.network.0xCAFE0004]
initial_peers = [
  "64.227.110.69",
  "188.166.3.140",
]

[node.network.0xCAFE0005]
initial_peers = [
  ...
]
```

Do you intend to work on it?

btw, are you on our Discord server? can you DM me?

**Comment by dan-da** [comment]
yeah, I can take a stab at it.   I'll sign up for discord later today.

**Comment by dan-da** [comment]
@steinerkelvin I sent you an email... discord signup failed.
